### PR TITLE
Faster build

### PR DESCRIPTION
**Pull request by ltratt**
These are a couple of changes from a few days back (used to build the last version of the experiment) that I had forgotten to make a PR for.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ generate Java source files from user input. Run `build_blackbox_data.sh` on
 the Blackbox server and, many hours later, you will have a `combos` file (which
 is the list of pairs needed to regenerate the source files) as well as the
 Java source in a `src_files` directory. **The latter must never be
-redistributed.** However, the `combos` file is safe is redistribution.
+redistributed.** However, the `combos` file is safe to redistribution.
```
Comment (ptersilie) [inline README.md]: `safe to redistribute`?

Comment (ltratt) [reply README.md]: Oops!

**Comment by ltratt** [comment]
OK, typo fixed.

**Comment by ptersilie** [comment]
Looks good. Wanna squash?

**Review by ptersilie — APPROVED** [review]

**Comment by ltratt** [comment]
OK, squashed.

**Pull request merged**